app/build.gradle

```diff
@@ -32,6 +32,7 @@ android {
 }
 
 dependencies {
+    implementation 'androidx.lifecycle:lifecycle-extensions:2.2.0'
```

Comment:
Why is there an emply line below. Dependencies should have an empty line after them only when the implementation type changes